File: cpp/Ejercicio_1.cpp
#include <iostream>

using namespace std;

int suma_arreglo(int arreglo[], int tamaño)
{
    int suma = 0;

    for (int i = 0; i < tamaño; i++)
    {
        suma += arreglo[i];
    }

    return suma;
}

int main()
{
    int numeros[] = {1, 2, 3, 4, 5};

    int resultado = suma_arreglo(numeros, 5);

    cout << "La suma es: " << resultado << endl;

    return 0;
}

File: java/Main1/src/Ejercicio_1/Ejercicio_1.java
public class Ejercicio_1 
{

    public static int sumaArreglo(int[] arreglo)
    {
        int suma = 0;

        for (int numero : arreglo)
        {
            suma += numero;
        }

        return suma;
    }

    public static void main(String[] args)
    {
        int[] numeros = {1, 2, 3, 4, 5};

        int resultado = sumaArreglo(numeros);

        System.out.println("La suma es: " + resultado);
    }
}
